### PR TITLE
Modal-view-component

### PR DESCRIPTION
**Pull request by qamarfj**
# Description

Created Modal View Component ,Modal will shown up when we hover over an element that is attached with Modal.

Fixes # (issue)
#42 

# How to test?

To test it please run "npm run storybook" and hover over a small box having text "hover over here".

# Checklist

- [x ] I have performed a self-review of my own code
- [x ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x ] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

**Comment by cph-kiwi** [comment]
Looks good @qamarfj.

**Review by cph-kiwi — APPROVED** [review]

**Review by TITHI1244 — APPROVED** [review]
Very nice code with easy-to-understand comments. Great job :) @qamarfj

File: src/client/components/Modal/Modal.css [hunk 1]
```diff
@@ -0,0 +1,34 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Mono:wght@500&display=swap');
```
Comment (cph-kiwi) [inline src/client/components/Modal/Modal.css:2]: I think @TITHI1244 has done a PR removing the need for these to be imported. Is that correct @TITHI1244 ?

**Review by cph-kiwi — APPROVED** [review]

**Review by aijazraja — APPROVED** [review]
your components looks good and working

File: src/client/components/Modal/Modal.css (hunk 1 above)
Comment (TITHI1244) [reply src/client/components/Modal/Modal.css:2]: Yes, that PR is ready now. So the imports won't be needed :)

File: src/client/components/Modal/Modal.css [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+}
+
+.modal-content {
+  width: 500px;
+  border-radius: 25px;
+  transition: all 0.3s ease-in-out;
+  font-family: 'Roboto', 'Roboto Mono', monospace;
+}
+
+.modal.show {
```
Comment (bytewiz) [inline src/client/components/Modal/Modal.css:23]:
```suggestion
.modal-show {
```

**Review by bytewiz — APPROVED** [review]
Seems good to me 👍

**Pull request merged**